### ISSUE TITLE
[Accessibility] ExpenseReport.Json: Screen Reader does not announce the name and state of the collapse down arrow icon buttons present under 'Amount' under 'Expense approval card'.

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6274]
### Target Platforms

UWP

### SDK Version

Application Version: 2.7.2108.2001

### Application Name

Visualizer

### Problem Description

[35357132](https://microsoft.visualstudio.com/OS/_workitems/edit/35357132)

Screen Reader does not announce the name and state of the collapse down arrow icon buttons present under 'Amount' under 'Expense approval card'. It just announces "Button".

### Card JSON

[ExpenseReport.json](https://github.com/microsoft/AdaptiveCards/blob/main/samples/v1.2/Scenarios/ExpenseReport.json)

**Comment by anna-dingler** [comment]
@jwoo-msft @paulcam206 Should we close this issue since ToggleVisibility is deprecated?

**Comment by anna-dingler** [comment]
Closing this issue because it is caused by ToggleVisibility which is a deprecated control.
